spec/listener_spec.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.describe Listener do
+  it "can be instanced" do
```

Comment:
Is this just a placeholder, or is it an actually useful thing to check?

Reply:
It's a placeholder.